test/integration/app/prompted_nobuild.js

```diff
@@ -26,6 +26,7 @@ var rimraf = require('rimraf');
 var fs = require('fs');
 
 var appGeneratorPath = path.join(__dirname, '../../../app');
+var extendedTimeout = 300000
```

Comment:
OK, that could be fine provided the network call does not cause the `npm run testintfast` to take more than a minute or so to run. If it does, then we should probably either tag the slow tests or move them to another file.